src/components/Login.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import '../style/Login.css';
+import { useHistory } from 'react-router-dom';
+import Logo from '../common/Logo';
+import TweetBtn from '../common/TweetBtn';
+import { login } from '../actions/loginActions'
+import { connect } from 'react-redux';
+
+
+function Login(props) {
+    const [name, setName] = useState('');
+    const [password, setPassword] = useState('');
+    const history = useHistory();
+
+    function handleNameChange(e) {
+        setName(e.target.value);
+    }
+    function handlePasswordChange(e) {
+        setPassword(e.target.value);
+    }
+
+    function handleLogin() {
+        try {
+            history.push('/home');
+            props.onLogin();
+        } catch (e) {
+            console.log(e);
+        }
+    }
+
+    return (
+        <div className="login-page">
+            <form className="login-form">
+                <div className="login-header">
+                    <Logo />
+                    <h2 className="header"> Log in to Twitter</h2>
```

Comment:
"Log in to Twitter" should be in string.js file that having in json all the strings.
It is kind of a preparation to localization.
Take a look how we do it at Gong's source code